### PR TITLE
VIH-8045 admin_web_sds

### PR DESCRIPTION
**Pull request by lowflying**
### JIRA link (if applicable) ###

VIH-8045

### Change description ###

SDS prep for admin-web

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by FarronF — APPROVED** [review]

File: charts/vh-admin-web/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 apiVersion: v2
 name: vh-admin-web
 home: https://github.com/hmcts/vh-admin-web
-version: 0.0.3
+version: 0.0.1
```
Comment (FarronF) [inline charts/vh-admin-web/Chart.yaml:5]: Should this be a lower version than before?

**Review by FarronF — APPROVED** [review]
This looks fine. Just refer to my comment about a version number if that's correct go ahead.

File: charts/vh-admin-web/Chart.yaml (hunk 1 above)
Comment (lowflying) [reply charts/vh-admin-web/Chart.yaml:5]: Yea, was just testing the helm chart pushing. Terrible news: it's not working.

**Review by chris-pateman — APPROVED** [review]
simples

**Pull request closed without merging**